File: 5423 - Acesso móvel a sistemas de informação/Classes_v2/src/Aluno.kt
open class Aluno: Pessoa {

    var turma: String

    constructor(nome: String, idade: Int, turma:String) : super(nome, idade) {

        this.turma = turma
        super.nome = nome
        super.idade = idade

    }


    override fun getInfo(): String {
        return super.getInfo()
    }

    fun MetodoAluno() {
        println("Aluno")
    }

}

File: 11027 - Desenvolvimento de aplicações móveis (plataforma iOS)/heranca_v1/heranca_v1/Aluno.swift
//
//  Aluno.swift
//  heranca_v1
//
//  Created by Gonçalo Feliciano on 24/06/2024.
//

import Foundation


class Aluno: Pessoa{
    
    var turma: String
    
    init(nome: String, idade: Int, email: String = "Sem email", turma: String = "Sem turma") {
        
        self.turma = turma
        
        
        super.init(nome: nome, idade: idade, email: email)
    }
    
    
    override func getInfo() -> String {
        "\(super.getInfo())\nTurma: \(turma)"
    }
    
    
}
